backend/openapi.json

```diff
@@ -820,6 +892,56 @@
           }
         }
       },
+      "NewPollingStationRequest": {
+        "type": "object",
+        "description": "Polling station of a certain [crate::election::Election]",
+        "required": [
+          "name",
+          "number",
+          "polling_station_type",
+          "street",
+          "house_number",
+          "postal_code",
+          "locality"
+        ],
+        "properties": {
+          "house_number": {
+            "type": "string"
+          },
+          "house_number_addition": {
+            "type": [
+              "string",
+              "null"
```

Comment:
Not sure why but there are no other `null`s in the file..

Reply:
Related question, should generating a new `openapi.ts` be part of this PR?

Reply:
Hmmm I thought we configured this in `lefthook`. @praseodym do you know if there's a reason we didn't? Or shall I add it?

Reply:
https://github.com/kiesraad/abacus/issues/544

Reply:
I think this can be fixed by annotating the `Option` fields in the `NewPollingStationRequest` struct with
```rust
    #[serde(skip_serializing_if = "Option::is_none")]
    #[schema(nullable = false)]
```

Reply:
> in the NewPollingStationRequest struct

Was checking the wrong struct indeed, thanks!